extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -131,118 +131,118 @@ public function adContextDataProvider() {
-			]],
+			] ],
 
-			$defaultParameters + ['expectedMappings' => [
+			$defaultParameters + [ 'expectedMappings' => [
 				'newVertical' => 'games',
 				'expectedMappedVertical' => 'gaming'
-			]],
+			] ],
 
-			$defaultParameters + ['expectedMappings' => [
+			$defaultParameters + [ 'expectedMappings' => [
 				'newVertical' => 'books',
 				'expectedMappedVertical' => 'ent'
-			]],
+			] ],
 
-			$defaultParameters + ['expectedMappings' => [
+			$defaultParameters + [ 'expectedMappings' => [
 				'newVertical' => 'comics',
 				'expectedMappedVertical' => 'ent'
-			]],
+			] ],
 
-			$defaultParameters + ['expectedMappings' => [
+			$defaultParameters + [ 'expectedMappings' => [
 				'newVertical' => 'lifestyle',
 				'expectedMappedVertical' => 'life'
-			]],
+			] ],
 
-			$defaultParameters + ['expectedMappings' => [
+			$defaultParameters + [ 'expectedMappings' => [
 				'newVertical' => 'not-existing',
 				'expectedMappedVertical' => 'error'
-			]],
+			] ],
 
-			$defaultParameters + ['expectedMappings' => [
+			$defaultParameters + [ 'expectedMappings' => [
 				'oldVertical' => 'Wikia',
 				'newVertical' => 'other',
-				'expectedMappedVertical' => 'wikia']
+				'expectedMappedVertical' => 'wikia' ]
 			],
 
-			array_merge($defaultParameters, [
+			array_merge( $defaultParameters, [
 				'expectedTargeting' => [ 'newWikiCategories' => [ 'test', 'test1' ] ],
-				'categories' => [ 'old' => ['test'], 'new' => ['test1'] ]
-			]),
+				'categories' => [ 'old' => [ 'test' ], 'new' => [ 'test1' ] ]
+			] ),
 
-			array_merge($defaultParameters, [
+			array_merge( $defaultParameters, [
 				'expectedTargeting' => [ 'newWikiCategories' => [ 0 => 'test', 2 => 'test1' ] ],
-				'categories' => [ 'old' => ['test'], 'new' => ['test', 'test1'] ]
-			]),
+				'categories' => [ 'old' => [ 'test' ], 'new' => [ 'test', 'test1' ] ]
+			] ),
 
-			array_merge($defaultParameters, [
+			array_merge( $defaultParameters, [
 				'expectedTargeting' => [ 'newWikiCategories' => [ 0 => 'test', 1 => 'test1', 4 => 'test2' ] ],
-				'categories' => [ 'old' => ['test', 'test1' ], 'new' => ['test', 'test1', 'test2'] ]
-			]),
+				'categories' => [ 'old' => [ 'test', 'test1' ], 'new' => [ 'test', 'test1', 'test2' ] ]
+			] ),
 
-			array_merge($defaultParameters, [
+			array_merge( $defaultParameters, [
 				'expectedTargeting' => [ 'newWikiCategories' => [ 'test2' ] ],
-				'categories' => [ 'old' => ['test2'] ]
-			]),
+				'categories' => [ 'old' => [ 'test2' ] ]
+			] ),
 
-			array_merge($defaultParameters, [
+			array_merge( $defaultParameters, [
 				'expectedTargeting' => [ 'newWikiCategories' => [ 'test3' ] ],
-				'categories' => [ 'new' => ['test3'] ]
-			]),
+				'categories' => [ 'new' => [ 'test3' ] ]
+			] ),
 		];
 	}
 
 	/**
 	 * Test getContext
 	 *
-	 * @covers AdEngine2ContextService::getContext
+	 * @covers       AdEngine2ContextService::getContext
 	 * @dataProvider adContextDataProvider
 	 */
 	public function testGetContext(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function has 9 parameters, which is greater than the 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS107)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function "testGetContext" has 175 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS138)